pkiintergration-fe/src/services/create-certificate.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { FetchClient } from '@c8y/client';
+import { AlertService, gettext } from '@c8y/ngx-components';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CreateKeyPairService {
+  constructor(private fetchClient: FetchClient, private alertService: AlertService) {}
+  async getKeyPair(deviceId: string): Promise<void> {
```

Comment:
suggestion: return fetch response

it might be helpful if the service return the response data to allow the consumption/handling based on the response in the component.